From f6ba5f6bc59690cb91ede18aacb1bc2ac392a738 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 24 Aug 2017 14:17:19 -0700
Subject: [PATCH] TOOLS-1807 node updates for july 2017 node.js security
 releases Reviewed by: Julien Gilli <julien.gilli@joyent.com> Approved by:
 Julien Gilli <julien.gilli@joyent.com>

---
 node.js/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/node.js/Makefile b/node.js/Makefile
index 9511c36..e8edf57 100644
--- a/node.js/Makefile
+++ b/node.js/Makefile
@@ -52,6 +52,7 @@ endif
 
 AUTOCONF_OPTS += \
 	--with-dtrace \
+	--without-snapshot \
 	--openssl-use-sys \
 	--openssl-libpath=$(DESTDIR)/lib \
 	--openssl-includes=$(DESTDIR)/usr/include \
-- 
2.21.0

